content/SCALETutorials/Credentials/BackupCredentials/AddSSHConnectionKeyPair.md

```diff
@@ -85,19 +85,19 @@ Using the **SSH Connections** configuration screen:
 
 {{< trueimage src="/images/SCALE/Credentials/NewSSHConnectAuthenticationManual.png" alt="Manual Authentication Settings" id="Manual Authentication Settings" >}}
 
-   a. Enter a host name or host IP address for the remote non-TrueNAS system as a valid URL.
+   a. Enter the host name or host IP address for the remote non-TrueNAS system as a valid URL.
    An IP address example is *https://10.231.3.76*.
```

Comment:
Are we sure this is correct? URL should not apply to non-TrueNAS, I would expect IP or hostname, no URL.

Reply:
@william-gr Should be "valid URL scheme", instead of "valid URL". Here's the tooltip in the UI.
![Screenshot 2024-09-12 135241](https://github.com/user-attachments/assets/c5d96c00-15d3-479c-9974-4e0487ad0e76)

Reply:
@william-gr Looks like that was copied from the semi-auto instructions. We will fix it. Thanks!